express_server_easyGpt/server.js

```diff
@@ -12,26 +12,26 @@ const OPENAI_API_KEY = process.env.OPENAI_API_KEY; // Your OpenAI API Key
 app.post("/askgpt", async (req, res) => {
   const model = req.body.model;
   const messages = req.body.messages;
-  
+
   let gpt = new EasyGpt();
-  gpt.setApiKey(OPENAI_API_KEY)
-  gpt.changeModel(model ?? "gpt-3.5-turbo")
+  gpt.setApiKey(OPENAI_API_KEY);
+  gpt.changeModel(model ?? "gpt-3.5-turbo");
 
-  messages.forEach(message => {
-    gpt.addMessage(message.content, message.role)
-  })
+  messages.forEach((message) => {
+    gpt.addMessage(message.content, message.role);
+  });
 
-  console.log(gpt)
+  // console.log(gpt);
```

Comment:
Probably makes more sense to remove this entirely as opposed to commenting it out?

Reply:
Could do!